src/EditorFeatures/Test2/Rename/InlineRenameTests.vb

```diff
@@ -767,6 +770,9 @@ End Class
                 Await WaitForRename(workspace)
                 Await VerifyTagsAreCorrect(workspace, "BarGoo")
                 Await VerifyNoRenameTrackingTags(renameTrackingTagger, workspace, document)
+
+                session.Commit()
+                Await VerifyTagsAreCorrect(workspace, "BarGoo", expectFileRename:=True)
```

Comment:
See other comment on VerifyTagsAreCorrect: seems like this should just be VerifyFileNameExists() or something, since there's no tags to verify after the session was committed.